libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/infopanel/InfoPanelBinder.kt

```diff
@@ -53,6 +54,17 @@ abstract class InfoPanelBinder : UIBinder {
      */
     abstract fun getContentLayout(layout: ViewGroup): ViewGroup?
 
+    /**
+     * Called when the Info Panel layout should apply system bar insets.
+     *
+     * This method should be overridden by a subclass to apply a policy different from the default behavior.
+     * The default behavior ignores insets.
+     *
+     * @param layout ViewGroup returned by [onCreateLayout]
+     * @param insets system bars insets
+     */
```

Comment:
Based on this comment https://github.com/mapbox/mapbox-navigation-android/pull/6145/files#r943294250 the sentence `The default behavior ignores insets` caused confusion for me. 
```suggestion
    /**
     * Called when the Info Panel layout should apply system bar insets.
     *
     * This method should be overridden by a subclass to apply a policy different from the default behavior.
     * The default behavior applies insets with a value of INSETS.NONE
     *
     * @param layout ViewGroup returned by [onCreateLayout]
     * @param insets system bars insets
     */
```